Sources/Redis/RedisID.swift

```diff
@@ -0,0 +1,32 @@
+import Foundation
+import Vapor
+
+public struct RedisID: Hashable, Codable {
```

Comment:
This should conform to the following protocols:


- RawRepresentable
- ExpressibleByStringLiteral
- ExpressibleByStringInterpolation
- CustomStringConvertible
- Comparable
- Hashable
- Codable

Reply:
```suggestion
/// A type-safe representation of a String representing individual identifiers of separate Redis connections and configurations.
///
/// It is recommended to define static extensions for your definitions to make it easier at call sites to reference them.
///
/// For example:
/// ```swift
/// extension RedisID { static let oceanic = RedisID("oceanic") }
/// app.redis(.oceanic) // Application.Redis instance
/// ```
public struct RedisID: Hashable, Codable {
```